src/libs/ReportActionsUtils.js

```diff
@@ -179,6 +180,34 @@ function getLastVisibleMessageText(reportID, actionsToMerge = {}) {
         .substring(0, CONST.REPORT.LAST_MESSAGE_TEXT_MAX_LENGTH);
 }
 
+/**
+ * Filter out any reportActions which should not be displayed
+ * Then, sort by their created timestamp first, and reportActionID second
+ * This is a combination of getSortedReportActions() and filterReportActionsForDisplay()
+ *
+ * @param {Object} reportActions
+ * @returns {Array}
+ */
+function getSortedReportActionsForDisplay(reportActions) {
+    // HACK ALERT: We're temporarily filtering out any reportActions keyed by sequenceNumber
+    // to prevent bugs during the migration from sequenceNumber -> reportActionID
+    const filteredReportActions = _.filter(reportActions, (reportAction, key) => {
+        if (!reportAction) {
+            return false;
+        }
+
+        if (String(reportAction.sequenceNumber) === key) {
+            Log.info('Front-end filtered out reportAction keyed by sequenceNumber!', false, reportAction);
+            return false;
+        }
+
+        return true;
+    });
+
+    const sortedReportActions = getSortedReportActions(filteredReportActions, true);
+    return filterReportActionsForDisplay(sortedReportActions);
+}
+
 export {
     getSortedReportActions,
     filterReportActionsForDisplay,
```

Comment:
I don't think this is referenced anywhere outside this file now, so it's export can be removed.

Reply:
I can see that both `getSortedReportActions` and `filterReportActionsForDisplay` are being used outside this file.
For `getSortedReportActions`: https://github.com/Expensify/App/blob/6f2705807eadea43972bc3fdef69237c8c24277a/src/libs/actions/User.js#L270
For `filterReportActionsForDisplay` https://github.com/Expensify/App/blob/6f2705807eadea43972bc3fdef69237c8c24277a/tests/unit/ReportActionsUtilsTest.js#L141

Reply:
Hi @tgolen Could you help to check my comment? Thanks